app/models/User.coffee

```diff
@@ -309,6 +309,7 @@ module.exports = class User extends CocoModel
   isOnPremiumServer: ->
     return true if me.get('country') in ['brazil']
     return true if me.get('country') in ['china'] and (me.isPremium() or me.get('stripe'))
+    return true if features?.china
```

Comment:
How does this overlap with the check just above?
Otherwise this looks good.

Reply:
I didn’t understand well about this problem - - I asked Nick about it before:

—I queried our China database, found that about 5000+ users country field is China, and others are null. When do we update country this field?
—from Nick:
because most users aren’t premium (they haven’t subscribed) or have paid via Stripe in the past
Country should have automatically via geoip on user creation, but for a while, API-created users weren’t getting country set, and if the geoip detection fails, it will be left blank also